src/test/java/subway/SubwayLineTest.java

```diff
@@ -0,0 +1,217 @@
+package subway;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.test.annotation.DirtiesContext;
+
+import io.restassured.RestAssured;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import subway.line.LineRequest;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@DisplayName("지하철 노선 관리 기능")
+@DirtiesContext(classMode = DirtiesContext.ClassMode.BEFORE_EACH_TEST_METHOD)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
+public class SubwayLineTest {
+
+	@BeforeEach
+	void createStations() {
+		createStation("신분당선");
+		createStation("새로운지하철역");
+		createStation("또다른지하철역");
+	}
+
+	/**
+	 * When 지하철 노선을 생성하면
+	 * Then 지하철 노선 목록 조회 시 생성한 노선을 찾을 수 있다
+	 */
+	@DisplayName("지하철 노선 생성")
+	@Test
+	void createLine() {
+		// when
+		LineRequest lineRequest = new LineRequest("신분당선", "bg-red-600", 1L, 2L, 10);
+		createLine(lineRequest);
+
+		// then
+		List<String> lineNames = showLines().jsonPath().getList("name", String.class);
+		assertThat(lineNames).containsAnyOf("신분당선");
+	}
+
+	/**
+	 * Given 2개의 지하철 노선을 생성하고
+	 * When 지하철 노선 목록을 조회하면
+	 * Then 지하철 노선 목록 조회 시 2개의 노선을 조회할 수 있다.
+	 * */
+	@DisplayName("지하철 노선 목록 조회")
+	@Test
+	void showLineList() {
+		// given
+		LineRequest lineRequest1 = new LineRequest("신분당선", "bg-red-600", 1L, 2L, 10);
+		LineRequest lineRequest2 = new LineRequest("분당선", "bg-green-600", 1L, 3L, 10);
+
+		createLine(lineRequest1);
+		createLine(lineRequest2);
+
+		// when
+		// then
+		List<String> lineNames = showLines().jsonPath().getList("name", String.class);
```

Comment:
```suggestion
		// when
		List<String> lineNames = showLines().jsonPath().getList("name", String.class);
		// then
```

when / then에 대해서 합치지 말고 각각 분리해서 나타내셨으면 좋을 것 같습니다 😊